### DISCUSSION TITLE
Is it possible to have all successfull PR's to combine?

### DISCUSSION THREAD

**Discussion by UnderKoen** [renovatebot/renovate#23350] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

We current target our development branch which has an expansive deploy ci. With renovate auto merge it creates a lot of deploys. Is it possible to combine all successful pr in another branch until the end of the day?

### What you tried so far.

I tried grouping but some dependecy upgrades brake the test's.

**Comment by rarkins** [comment, marked as answer]
Duplicate of #4404

**Comment by HonkingGoose** [comment]
> We current target our development branch which has an expansive deploy ci. With renovate auto merge it creates a lot of deploys. Is it possible to combine all successful pr in another branch until the end of the day?

I see you're using the Mend Renovate app. Renovate support GitHub's Merge Queue, did you try that? Here's what you need to use Merge Queue on your repository: [^gh-docs]

> Pull request merge queues are available in any public repository owned by an organization, or in private repositories owned by organizations using GitHub Enterprise Cloud. For more information, see "[GitHub’s products](https://docs.github.com/en/get-started/learning-about-github/githubs-products)."

Here are the relevant docs:

- [Renovate docs, key concepts: GitHub Merge Queue](https://docs.renovatebot.com/key-concepts/automerge/#github-merge-queue)
- [GitHub Docs about Merge Queues](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue)

[^gh-docs]: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue
